module/entities/helper.js

```diff
@@ -318,50 +294,56 @@ module.exports = class EntitiesHelper {
         return new Promise(async (resolve, reject) => {
 
             try {
-
-                let result = [];
+                
+                let result = {};
+                let subEntitiesResult = {};
+                let uniqueEntities = [];
```

Comment:
@VISHNUDAS-tunerlabs rename this to listOfImmediateEntities.

Reply:
@aks30 uniqueEntities -> can be immediate entities or entities find via traversal . so renaming listOfSubEntities looks more suitable.